pilot/pkg/proxy/envoy/v2/cds.go

```diff
@@ -78,7 +78,14 @@ func (s *DiscoveryServer) generateRawClusters(node *model.Proxy, push *model.Pus
 		return nil, err
 	}
 
+	if sdsTokenPath, found := node.Metadata[model.NodeMetadataSdsTokenPath]; found && len(sdsTokenPath) > 0 {
+		// If SDS_TOKEN_PATH is in the node metadata, make a copy of rawClusters so that
+		// the path of SDS token will be applied to the copied clusters.
+		rawClusters = CopyClusters(rawClusters)
```

Comment:
You dont have to do this copy clusters stuff anymore. WE dont have a CDS cache. So you can make these changes in your auth plugin directly.

Reply:
https://github.com/istio/istio/pull/12304/files

Reply:
Thanks! Revised the PR to use the pilot.EnableCDSPrecomputation() flag such that this copy clusters stuff is only done when pilot.EnableCDSPrecomputation() is true.